includes/class-wc-gateway-stripe.php

```diff
@@ -1162,4 +1163,86 @@ public function validate_account_statement_descriptor_field( $param, $value, $ma
 
 		return $value;
 	}
+
+	/**
+	 * Get required setting keys for setup.
+	 *
+	 * @return array Array of setting keys used for setup.
+	 */
+	public function get_required_settings_keys() {
+		return array( 'publishable_key', 'secret_key' );
+	}
+
+	/**
+	 * Get the connection URL.
+	 *
+	 * @return string Connection URL.
+	 */
+	public function get_connection_url( $return_url = '' ) {
+		$api     = new WC_Stripe_Connect_API();
+		$connect = new WC_Stripe_Connect( $api );
+
+		$url = $connect->get_oauth_url( $return_url );
+
+		return is_wp_error( $url ) ? null : $url;
+	}
+
+	/**
+	 * Get help text to display during quick setup.
+	 *
+	 * @return string
+	 */
+	public function get_setup_help_text() {
+		return sprintf(
+			__( 'Your API details can be obtained from your <a href="%s">Stripe account</a>. Don’t have a Stripe account? <a href="%s">Create one.</a>', 'woocommerce-gateway-stripe' ),
+			'https://stripe.com/docs/keys',
```

Comment:
Based on the context of the text, it appears that the intended URL for the first string should be `https://dashboard.stripe.com/apikeys`, as opposed to `https://stripe.com/docs/keys`.

Is this correct? Or, should the context of the text be updated to indicate that the link will navigate the user to the Stripe docs, instead?

Reply:
CC @joshuatf

Reply:
I believe that is correct, @lovo-h!